uaclient/config.py

```diff
@@ -276,12 +279,47 @@ def _remove_beta_resources(self, response) -> "Dict[str, Any]":
 
         return new_response
 
+    def _get_config_status(self) -> "Dict[str, str]":
+        """Return a dict with configStatus and configStatusDetails keys.
+
+            Values for configStatus will be one of UserFacingConfigStatus enum:
+                inactive, active, reboot-required
+            configStatusDescription will provide more details about that state.
+        """
+        userStatus = status.UserFacingConfigStatus
+        status_val = userStatus.INACTIVE.value
+        status_desc = status.MESSAGE_NO_ACTIVE_OPERATIONS
+        if os.path.exists(self.data_path("lock")):
+            lock_file = self.data_path("lock")
+            lock_content = util.load_file(lock_file)
+            [lock_pid, lock_holder] = lock_content.split(":")
```

Comment:
This part of the code seems really similar to the one we have in lock decorator, maybe we can move it to util module

Reply:
moved it to util.check_lock_info for use in both callsites.